zerver/views/saved_snippets.py

```diff
@@ -67,9 +65,6 @@ def edit_saved_snippet(
         ),
     ] = None,
 ) -> HttpResponse:
-    if title is None and content is None:
-        raise JsonableError(_("No new data is supplied"))
```

Comment:
I think it might be better to do a `return json_success` here, to avoid the side effect of sending a noop change event.